String support

Hi, maybe is a my mistake, but i cant use the String type with 'stdout', only the str type.

        let mut main = Command::main_binary().unwrap();

        main.arg("tags")
            .arg("list")
            .arg("--use-csv");

        let stdout = String::from("\"Name\",\"#id\"");

        // stdout.push_str ....
        // stdout.push_str ....
        // stdout.push_str ....
      
        main.assert()
            .success()
            .stdout(stdout.as_str());

On run this, i receive `error[E0597]: stdout does not live long enough`. I cant use `.stdout(stdout.as_str());`, because i receive `the trait assert_cmd::assert::IntoOutputPredicate<_> is not implemented for std::string::String`.

Where am I going wrong :( ?

Thank you in advance for your attention.

--- a/src/assert.rs
+++ b/src/assert.rs
@@ -819,7 +819,12 @@ pub struct StrContentOutputPredicate(
 );
 
 impl StrContentOutputPredicate {
-    pub(crate) fn new(value: &'static str) -> Self {
+    pub(crate) fn from_str(value: &'static str) -> Self {
+        let pred = predicates::str::similar(value).from_utf8();
+        StrContentOutputPredicate(pred)
+    }
+
+    pub(crate) fn from_string(value: String) -> Self {
         let pred = predicates::str::similar(value).from_utf8();
         StrContentOutputPredicate(pred)
     }
@@ -858,11 +863,19 @@ impl fmt::Display for StrContentOutputPredicate {
     }
 }
 
+impl IntoOutputPredicate<StrContentOutputPredicate> for String {
+    type Predicate = StrContentOutputPredicate;
+
+    fn into_output(self) -> Self::Predicate {
+        Self::Predicate::from_string(self)
+    }
+}
+
 impl IntoOutputPredicate<StrContentOutputPredicate> for &'static str {
     type Predicate = StrContentOutputPredicate;
 
     fn into_output(self) -> Self::Predicate {
-        Self::Predicate::new(self)
+        Self::Predicate::from_str(self)
     }
 }
 

--- a/tests/assert.rs
+++ b/tests/assert.rs
@@ -57,3 +57,14 @@ fn stderr_example() {
         .assert()
         .stderr("world\n");
 }
+
+#[test]
+fn stdout_string() {
+    let expected = "hello\n".to_owned();
+    Command::main_binary()
+        .unwrap()
+        .env("stdout", "hello")
+        .env("stderr", "world")
+        .assert()
+        .stdout(expected);
+}

